foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -177,7 +177,7 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   Another option for opening your HTML pages in the browser is using the [live server extension](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) with VSCode. This will open your HTML document and automatically refresh it every time you save the document. However, we recommend not using this extension and instead doing it the old-fashioned way, by opening the page and refreshing the page manually in the browser for your first few HTML projects. This way, you can get used to that process and won't become reliant on extensions right away. One reason is that there may be subtle differences when using extensions. For example, live server will always use UTF-8 character encoding and not the value defined in your `meta-charset` attribute. This could potentially hide some characters on your site as they will not be encoded the way you expect.
+*   Another option for opening your HTML pages in the browser is using the [live server extension](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) with VSCode. This will open your HTML document and automatically refresh it every time you save the document. However, we recommend not using this extension and instead doing it the old-fashioned way, by opening the page and refreshing the page manually in the browser for your first few HTML projects. This way, you can get used to that process and won't become reliant on extensions right away. One reason is that there may be subtle differences when using extensions. For example, live server will always use UTF-8 character encoding and not the value defined in your `<meta charset>` element. This could potentially hide some characters on your site as they will not be encoded the way you expect.
```

Comment:
In addition to just using `<meta>` element in this line as well.

Reply:
Following your earlier explanation, I think one should write here

> For example, live server will always use UTF-8 character encoding and not the value defined in your `<meta>` element.

Reply:
What do you think about adding the `charset` here?
> For example, live server will always use UTF-8 character encoding and not the `charset` value defined in your `<meta>` element.

Reply:
Yes, excellent.

Reply:
Great! I think this change is the only thing still needed on this PR before I can approve it.

Reply:
Done. (At least I hope so.)